backend/redis/utils.go

```diff
@@ -65,6 +64,10 @@ func ServiceGithubErrorCountKey(serviceId int) string {
 	return fmt.Sprintf("service-github-errors-%d", serviceId)
 }
 
+func GithubRateLimitKey(gitHubRepo string) string {
+	return fmt.Sprintf("github-rate-limit-exceeded-%s", gitHubRepo)
```

Comment:
Update rate limit to be dependent on github repo name since the rate limit is set per repo installation token